### ISSUE TITLE
Add feed correlation to the graph (toggle view)

### ISSUE THREAD

**Issue by adulau** [MISP/MISP#2221]
Add feed correlation to the graph (toggle view)

**Comment by iglocska** [comment]
To the correlation graph? Good idea.

**Comment by adulau** [comment]
Yep the correlation graph to have a toggle with or without feed correlation. I suppose this can be pretty fast with the feed Redis backend ;-)
